File: Api/Api/Api/Model/Clientes.cs


namespace Api.Model
{
    public class Clientes
    {
        public int id { get; set; }
        public string Nome { get; set; }
        public string Documento { get; set; }
        public string Cidade { get; set; }
        public string Endereco { get; set; }
        public string Complemento { get; set; }
        public int idLocalizacao { get; set; }

        //public virtual Localizacao Localizacao { get; set; }


    }
}

File: Front/src/app/Model/Clientes.ts
export class Clientes {
    id: any;
    Nome: any;
    Documento: any;
    Cidade: any;
    Endereco: any;
    Complemento: any;
    idLocalizacao: any;
}

export class DataReturn<T>
{
    status: string;
    data: T;
}